### PR TITLE
chore(POINT-941): Add bancontact to JS

### PR DESCRIPTION
**Pull request by pablo-diaz-cko**
## Description
We want to add bancontact to our JS so it's shown in the front-end. The issue we’re trying to solve is that currently, the merchant is unaware a card is Bancontact - therefore if a customer wrongly selects this, it could lead to failures.

## Solution

Add these values for bancontact:
```
6703 
487104
670305
670330
670319
670300
670342
487109
670397
670375
670310
```
Screenshot 2023-05-09 at 16 46 40

## Jira Issue
https://checkout.atlassian.net/browse/POINT-941

### CODE REVIEW

**Comment by pablo-diaz-cko** [comment]
@processout/frontend-developers Can I have a review here, please?

**Review by wojciech-niedbala-cko — COMMENTED** [review]
I believe you need to run `make build` as well

**Comment by pablo-diaz-cko** [comment]
> I believe you need to run `make build` as well

Done: 2b05132a642b17ca05718c710be673c1dd864e6a, Thanks!

**Review by adam-blevins-cko — APPROVED** [review]
LGTM!

The attached ticket talked about updating this switch statement for card number lengths based on scheme. There is no `carte bancaire` case, nor any new one for the `bancontact` case. 

Do we know if that's what we want? Are the defaults for min/max length of the card number good enough here (min 12, max 19)?

**Comment by pablo-diaz-cko** [comment]
> LGTM!
> 
> The attached ticket talked about updating [this switch statement](https://github.com/processout/processout.js/blob/master/src/processout/card.ts#L568) for card number lengths based on scheme. There is no `carte bancaire` case, nor any new one for the `bancontact` case.
> 
> Do we know if that's what we want? Are the defaults for min/max length of the card number good enough here (min 12, max 19)?

We will raise a separate PR for that.

**Comment by zhengyi-li-cko** [comment]
I guess the pin is same as normal card, 3 digits, right?

**Comment by pablo-diaz-cko** [comment]
I have attached a test Screenshot 2023-05-09 at 16 48 33

File: src/processout/card.ts [hunk 1]
```diff
@@ -661,8 +661,8 @@ module ProcessOut {
...
+                "bancontact": ["6703", "487104", "670305", "670330", "670319", "670300", "670342", "487109", "670397", "670375", "670310"]
```
Comment (pierre-lafanechaire-cko) [inline src/processout/card.ts:665]: Adding `6703` and `670305` (and all the others) is redundant. As shared enclosed with a Bancontact BIN range analysis, we may be able to clean the code by just having the Maestro range (`6703`) and Visa (`487104`)
[bancontact.csv](https://github.com/processout/processout.js/files/11438589/bancontact.csv)

**Pull request merged**